sdk/storage/azure-storage-blob/apilisting.txt

```diff
@@ -0,0 +1,591 @@
+	def send(self, request: request) -> ~azure.core.pipeline.PipelineResponse: ...
+	def sleep(self, settings, transport): ...
+
+class LocationMode(object):
+	...
+
+class Logging(Model, object):
+	version:str
+	delete:bool
+	read:bool
+	write:bool
+	retention_policy:retention_policy
+
+	def __eq__(self, other): ...
+	def __init__(self, *, version, delete, read, write, retention_policy, **kwargs): ...
+	def __ne__(self, other): ...
+	def __str__(self): ...
+	def as_dict(self, keep_readonly=True, key_transformer: function=<function attribute_transformer at 0x10e6a4510>) -> dict: ...
+	@classmethod
+	def deserialize(cls, data: str, content_type: str=None): ...
+	@classmethod
+	def enable_additional_properties_sending(cls): ...
+	@classmethod
+	def from_dict(cls, data: dict, key_extractors=None, content_type: str=None): ...
+	@classmethod
+	def is_xml_model(cls): ...
+	def serialize(self, keep_readonly: bool=False) -> dict: ...
+	def validate(self) -> list: ...
+
+class Metrics(Model, object):
+	version:str
+	enabled:bool
+	include_apis:bool
+	retention_policy:retention_policy
+
+	def __eq__(self, other): ...
+	def __init__(self, *, version, enabled, include_apis, retention_policy, **kwargs): ...
+	def __ne__(self, other): ...
+	def __str__(self): ...
+	def as_dict(self, keep_readonly=True, key_transformer: function=<function attribute_transformer at 0x10e6a4510>) -> dict: ...
+	@classmethod
+	def deserialize(cls, data: str, content_type: str=None): ...
+	@classmethod
+	def enable_additional_properties_sending(cls): ...
+	@classmethod
+	def from_dict(cls, data: dict, key_extractors=None, content_type: str=None): ...
+	@classmethod
+	def is_xml_model(cls): ...
+	def serialize(self, keep_readonly: bool=False) -> dict: ...
+	def validate(self) -> list: ...
+
+class NoRetry(AsyncStorageRetryPolicy, StorageRetryPolicy, HTTPPolicy, ABC, Generic, object):
+	def __class_getitem__(cls, params): ...
+	def __init__(self): ...
+	def __init_subclass__(cls, *args, **kwargs): ...
+	def __new__(cls, *args, **kwargs): ...
+	def configure_retries(self, request): ...
+	def get_backoff_time(self, settings) -> float: ...
+	def increment(self, *args, *, response, error, **kwargs): ...
+	def send(self, request: request) -> ~azure.core.pipeline.PipelineResponse: ...
+	def sleep(self, settings, transport): ...
+
+class PageRange(DictMixin, object):
+	start:int
+	end:int
+
+	def __init__(self, start: int=None, end: int=None): ...
+	def __str__(self): ...
+	def get(self, key, default=None): ...
+	def has_key(self, k): ...
+	def items(self): ...
+	def keys(self): ...
+	def update(self, *args, **kwargs): ...
+	def values(self): ...
+
+class PremiumPageBlobTier(str, Enum, object):
+	...
+
+class PublicAccess(str, Enum, object):
+	...
+
+class ResourceTypes(object):
+	ResourceTypes.CONTAINER:ResourceTypes
+	ResourceTypes.OBJECT:ResourceTypes
+	ResourceTypes.SERVICE:ResourceTypes
+	service:bool
+	container:bool
+	object:bool
+	_str:str
+
+	def __add__(self, other): ...
```

Comment:
`_add_` and `_or_` are doing the same thing, 

breaking change: remove enum like parameters